### PR TITLE
update multiselect error message

### PR DESCRIPTION
**Pull request by thanhz**
image

### CODE REVIEW

File: test/controllers/AddAdditionalMetadataControllerSpec.scala [hunk 1]
```diff
@@ -296,7 +296,7 @@ class AddAdditionalMetadataControllerSpec extends FrontEndTestHelper {
       )
       addAdditionalMetadataPageAsString should include("""    <p class="govuk-error-message" id="error-FoiExemptionCode">
           |        <span class="govuk-visually-hidden">Error:</span>
-          |        Search for and select at least one FOI exemption code(s)
+          |        Select at least one FOI exemption code(s)
```
Comment (TomJKing) [inline test/controllers/AddAdditionalMetadataControllerSpec.scala:299]: Is the `...(s)` needed on the word `code`. Make sense without it

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor comment. Thanks

File: test/controllers/AddAdditionalMetadataControllerSpec.scala (hunk 1 above)
Comment (thanhz) [reply test/controllers/AddAdditionalMetadataControllerSpec.scala]: I think this was based on an old copy change. But makes sense to remove it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**